### ISSUE TITLE
urlpatterns marked as "Invalid constant name"

### ISSUE THREAD

**Issue by webmaster128** [landscapeio/pylint-django#90]
I can confirm the issue in the now closed #41: pylint with pylint-django enabled marks the Django constant name `urlpatterns` as invalid.

My `mymodule/urls.py ` is

``` 
from django.conf.urls import url
from . import views

urlpatterns = [
    url(r'^$', views.index, name='index'),
]
```

and the error

```
$ PYTHONPATH=. pylint --load-plugins pylint_django mymodule/urls.py 
************* Module patches.urls
C:  4, 0: Invalid constant name "urlpatterns" (invalid-name)
```

All other files are linted fine, so I guess my setup is okay.

```
// Python 3.5.2
Django==1.10.5
...
pylint==1.6.5
pylint-django==0.7.2
pylint-plugin-utils==0.2.4
```

**Comment by charlydelta** [comment]
I have the same problem (Though with Django 1.11.x and pylint 1.7.2)

**Comment by atodorov** [comment]
Fixed in #109, #113
